lib/db-manager.js

```diff
@@ -132,6 +133,41 @@ var dbManager = module.exports = {
       });
    },
 
+   updateCommitCheck: function(body) {
+      let repo = body.repository.full_name;
+      let sha = body.check_run.head_sha;
+      let state = utils.mapCheckToStatus(body.check_run.conclusion || body.check_run.status);
+      let desc = utils.mapCheckToStatus(body.check_run.conclusion || body.check_run.status);
```

Comment:
Did you intend these to be identical?  maybe `let desc = state` instead so that's explicit?

Reply:
Yeah. Check runs don't really have a description like status checks do so I just set the description to state.

I can make that change.

Reply:
```suggestion
      let desc = state;
```